### ISSUE TITLE
Pulsus paradoxus

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10132]
**Preferred term label:**
Pulsus paradoxus

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0232118

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Pericarditis, constrictive

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000046
- label: Pulsus paradoxus
- parents: [Abnormal pulse (HP:0032552)](https://hpo.jax.org/app/browse/term/HP:0032552)
- pmid: PMID:29493917
- def: Pulsus paradoxus refers to a systolic pressure drop greater than 10mmHg during inspiration.
- comment: Pulsus paradoxus results from alterations in the mechanical forces imposed on the chambers of the heart and pulmonary vasculature and is often due to pericardial disease, particularly cardiac tamponade and to a lesser degree constrictive pericarditis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10132
- synonyms: Kussmaul pulse(NONE)|Paradoxical pulse volume(NONE)|Paradoxical pulse(NONE)
